.copier-answers.yml

```diff
@@ -1,7 +1,8 @@
 # Do NOT update manually; changes here will be overwritten by Copier
-_commit: v1.16.0
+_commit: v1.19.1
 _src_path: https://github.com/OCA/oca-addons-repo-template.git
 ci: GitHub
+convert_readme_fragments_to_markdown: false
```

Comment:
@AungKoKoLin1997 What's your opinion about this?  Community is leaning towards using .md files, I believe.  Should we make a switch?

Reply:
I don't see remarkable pros and crons from changing to markdown from rst. I think OCA makes this adjustment for non technical contributors as per note from Stéphane Bidoul and will not impact on our organization.

> * The readme fragments are converted to markdown. This makes it easier for non-technical contributors, and is also more robust (wrt subtitles formats, for instance). This is already reflected in the OCA module template

Whatever, we should try to switch to markdown because there is no disadvantages, it is more simple and easier.